homework_04/11_liliya_lyubenova/problem.py

```diff
@@ -0,0 +1,44 @@
+"""Find relative position for 2 circles"""
```

Comment:
I personally prefer the quotes to be on different lines for better readability but this way is also fine.

Reply:
Thanks, you are right. I will correct them.